app/src/main/kotlin/edu/stanford/bdh/engagehf/health/symptoms/SymptomsScreen.kt

```diff
@@ -280,12 +271,12 @@ fun SymptomsChart(
 }
 
 @Composable
-fun SymptomsDropdown(headerData: HeaderData, onAction: (SymptomsViewModel.Action) -> Unit) {
+private fun SymptomsDropdown(headerData: HeaderData, onAction: (SymptomsViewModel.Action) -> Unit) {
     Box(modifier = Modifier.wrapContentSize(Alignment.TopStart)) {
         TextButton(onClick = {
             onAction(SymptomsViewModel.Action.ToggleSymptomTypeDropdown(true))
         }) {
-            SymptomTypeText(headerData.selectedSymptomType)
+            SymptomTypeTitleText(headerData.selectedSymptomType)
```

Comment:
It would be great to expose a `selectedSymptomTypeText: StringResource` via the header data. The benefit is that we don't need to recalculate it as it is being done in `SymptomTypeTitleText` on every recomposition. You can then unit test the mapper.